mi-scheduler/base/argo-workflows/mi-workflow-template.yaml

```diff
@@ -0,0 +1,141 @@
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: mi
+  annotations:
+    thoth-station.ninja/template-version: 0.1.0
+  labels:
+    app: mi
+
+spec:
+  serviceAccountName: argo
+  entrypoint: mi-workflow
+
+  templates:
+    - name: mi-workflow
+      inputs:
+        parameters:
+          - name: REPOSITORY
+          - name: IMAGE_STREAM_REGISTRY
+          - name: IMAGE_STREAM_NAMESPACE
+          - name: IMAGE_STREAM_TAG
+      steps:
+        - - name: collect-knowledge
+            template: analyse
+            arguments:
+              parameters:
+                - name: REPOSITORY
+                  value: "{{inputs.parameters.REPOSITORY}}"
+                - name: IMAGE_STREAM_REGISTRY
+                  value: "{{inputs.parameters.IMAGE_STREAM_REGISTRY}}"
+                - name: IMAGE_STREAM_NAMESPACE
+                  value: "{{inputs.parameters.IMAGE_STREAM_NAMESPACE}}"
+                - name: IMAGE_STREAM_TAG
+                  value: "{{inputs.parameters.IMAGE_STREAM_TAG}}"
+
+        - - name: process-information
+            template: process
+            arguments:
+              parameters:
+                - name: REPOSITORY
+                  value: "{{inputs.parameters.REPOSITORY}}"
+                - name: IMAGE_STREAM_REGISTRY
+                  value: "{{inputs.parameters.IMAGE_STREAM_REGISTRY}}"
+                - name: IMAGE_STREAM_NAMESPACE
+                  value: "{{inputs.parameters.IMAGE_STREAM_NAMESPACE}}"
+                - name: IMAGE_STREAM_TAG
+                  value: "{{inputs.parameters.IMAGE_STREAM_TAG}}"
+
+    - name: analyse
+      inputs:
+        parameters:
+          - name: REPOSITORY
+          - name: IMAGE_STREAM_REGISTRY
+          - name: IMAGE_STREAM_NAMESPACE
+          - name: IMAGE_STREAM_TAG
+      container:
+        image: "{{inputs.parameters.IMAGE_STREAM_REGISTRY}}/\
+        {{inputs.parameters.IMAGE_STREAM_NAMESPACE}}/\
+        srcopsmetrics:{{inputs.parameters.IMAGE_STREAM_TAG}}"
```

Comment:
```suggestion
        image: "mi:latest"
```

Reply:
As the image are no longer kept in different namespace, we dont need to provide, image_registry, image_namespace,etc. please remove those variables in the workflow.